imkar/scattering/scattering.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import pyfar as pf
+from imkar import utils
+
+
+def freefield(sample_pressure, reference_pressure, microphone_weights):
+    r"""
+    Calculate the free-field scattering coefficient for each incident direction
+    using the Mommertz correlation method [1]_:
+
+    .. math::
+        s(\vartheta_S,\varphi_S) = 1 -
+            \frac{|\sum \underline{p}_{sample}(\vartheta_R,\varphi_R) \cdot
+            \underline{p}_{reference}^*(\vartheta_R,\varphi_R) \cdot w|^2}
+            {\sum |\underline{p}_{sample}(\vartheta_R,\varphi_R)|^2 \cdot w
+            \cdot \sum |\underline{p}_{reference}(\vartheta_R,\varphi_R)|^2
+            \cdot w }
+
+    with the ``sample_pressure``, the ``reference_pressure``, and the
+    area weights ``weights_microphones``. See
+    :py:func:`random_incidence` to calculate the random incidence
+    scattering coefficient.
+
+    Parameters
+    ----------
+    sample_pressure : pyfar.FrequencyData
+        Reflected sound pressure or directivity of the test sample. Its cshape
+        needs to be (..., #microphones).
```

Comment:
'#microphones' is a bit informal. Should we use `variable_name.size`? Would also affect the remaining docstrings.